pom.xml

```diff
@@ -433,6 +437,30 @@
 				</executions>
 			</plugin>
 
+			<!--
+				See
+				https://maven.apache.org/guides/mini/guide-reproducible-builds.html,
+				https://maven.apache.org/plugins/maven-artifact-plugin/buildinfo-mojo.html#referenceRepo.
+			-->
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-artifact-plugin</artifactId>
+				<version>3.1.0</version>
+				<configuration>
+					<reproducible>false</reproducible>
```

Comment:
Why false? As far as i understood setting it to true would drop environment information that we do not use anyhow.

Reply:
Indeed, we don't use that info right now. Ideally, it would be deployed together with the other artifacts. I just thought to keep this little bit of extra info for the time being.

Reply:
The plugin's `detectSkip` option (default `true`) makes that only deployed JARs are checked. We already skipped the deployment of some JARs many weeks back (by adding the property `maven-deploy-skip`), that is why it is not reflected in this PR. In a nutshell, the idea is to only deploy those artifacts to Maven Central that are used by developers (plugins, CLI and upstream modules). Other components like `rest-backend` are better consumed by downloading the Docker images. Nevertheless, we can extend the check also to those JARs, I will look at this...